Developer_Contributed/01-Versal_Custom_Thin_Platform_Extensible_System/linux/petalinux/Makefile

```diff
@@ -49,7 +49,7 @@ help:
 
 all: ${PROJ_DIR} 
 
-${PROJ_DIR}: ${XSA_DIR} 
+${PROJ_DIR}: ${XSA_DIR}/*.xsa
```

Comment:
`*.xsa` in Makefile list may not really mean all files with XSA suffix. 

Reference:
- https://stackoverflow.com/questions/40155241/how-to-properly-use-or-asterisk-symbols-as-a-placeholder-in-makefile
- https://www.gnu.org/software/make/manual/html_node/Wildcard-Function.html#Wildcard-Function